cmd/skipper/main.go

```diff
@@ -70,6 +71,8 @@ const (
 	accessLogUsage                 = "output file for the access log, When not set, /dev/stderr is used"
 	accessLogDisabledUsage         = "when this flag is set, no access log is printed"
 	debugEndpointUsage             = "when this address is set, skipper starts an additional listener returning the original and transformed requests"
+	certPathTLSUsage               = "path of the certificate file"
+	keyPathTLSUsage                = "path of the key"
```

Comment:
I'm not sure if it should be "path of" or "path to", my English is failing me here, but I see that in other places in skipper that "path to" is used so I'll change it. Moreover, I would like to understand if this description is enough (according to you) given that is fairly shorter than other usage descriptions in skipper.